content/applications/websites/livechat.rst

```diff
@@ -5,80 +5,287 @@
+pop-up.
+
+.. image:: livechat/create-rules.png
+   :align: center
+   :alt: View of a channel's rules form for Odoo Live Chat.
+
+Create new rules
+~~~~~~~~~~~~~~~~
+
+Fill out the fields on the :guilabel:`Open: Rules` pop-up as instructed below, then click
+:guilabel:`Save`.
+
+.. tabs::
+
+   .. tab:: Live Chat Button
 
-External options
-----------------
+      The *Livechat Button* is the icon that appears in the bottom corner of the website. Select
+      from one of the following display options:
 
-| If your website was not created with Odoo, you can find the code to be added to your own, under
-  the *Widget* tab.
-| Odoo also offers an URL you can send to users so they can have access to a single live chat page.
+      - :guilabel:`Show` displays the chat button on the page(s).
+      - :guilabel:`Show with notification` displays the chat button, as well as a floating text
+        bubble next to the button.
+      - :guilabel:`Open automatically` displays the button and automatically opens the chat window
+        after a specified amount of time (designated in the :guilabel:`Open automatically timer`
+        field).
+      - :guilabel:`Hide` hides the chat button on the page(s).
 
-.. image:: livechat/widget.png
-   :alt: View of the widget tab for Odoo Live Chat
+   .. tab:: Chatbot
 
-Managing chat requests
-======================
+      If a :guilabel:`Chatbot` will be included on this channel, select it from the dropdown. If the
+      chatbot will only be active when no operators are active, check the box labeled
+      :guilabel:`Enabled only if no operator`.
 
-Conversations initiated by visitors pop up as a direct message, and are shown in *Discuss*.
-Therefore, inquiries can be answered wherever you are in Odoo.
+   .. tab:: URL Regex
+
+      In the :guilabel:`URL Regex` field, input the relative URL of the page where the chat button
+      should appear.
+
+   .. tab:: Open automatically timer
+
+      This field designates the amount of time (in seconds) a page will be open before the chat
+      window will open. If the :guilabel:`Livechat Button` for this rule is not set to
+      :guilabel:`Open automatically`, this field will be ignored.
+
+   .. tab:: Country
+
+      If this channel should only be available to site visitors in specific countries, add them to
+      the :guilabel:`Country` field. If this field is left blank, the channel will be available to
+      all site visitors, regardless of location.
 
-.. image:: livechat/discuss.png
-   :alt: View of the discuss application with a message sent through live chat in Odoo
 
 .. note::
-   Conversations are dispatched based on the current workload of the online operators.
+   In order to track the geographical location of visitors, :guilabel:`GeoIP` must be installed on
+   the database. While this feature is installed by default on :guilabel:`Odoo Online`,
+   :guilabel:`On-Premise` databases will require additional :doc:`setup steps
```

Comment:
I dont think we should add a guilable to the database types, italics is probably best